libmpcdec/huffman.c

```diff
@@ -54,7 +54,7 @@ static const mpc_huffman mpc_table_HuffDSCF [16] = {
 	{0x8400, 6, 7}, {0x8000, 6, -7}, {0x6000, 3, -1}, {0x4000, 3, 2},
 	{0x3000, 4, 4}, {0x2800, 5, 6}, {0x2000, 5, -6}, {0x0, 3, -2}
 };
-mpc_lut_data mpc_HuffDSCF = {mpc_table_HuffDSCF, {}};
+mpc_lut_data mpc_HuffDSCF = {mpc_table_HuffDSCF, {{0}}};
```

Comment:
I need to validate this under Windows, last time I checked it wanted to initialize all the members explicitly.

Reply:
Looks good, MSVC approves.